02.calendar/calender.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+WIDE_LENGTH = 20 #カレンダーの横幅の文字数
+
+year = Date.today.year
+month = Date.today.month
+
+opt = OptionParser.new
+opt.on('-y [VAL]') {|v| year = v.to_i }
+opt.on('-m [VAL]') {|v| month = v.to_i }
+opt.parse(ARGV)
+
+start_date = Date.new(year, month, 1)
+last_date = Date.new(year, month, -1)
+
+#年と月表示処理
+date = Date.new(year, month, 1)
+month_name = date.strftime("%B")
+hedding = month_name + " " + year.to_s
+puts hedding.center(WIDE_LENGTH)
+
+#曜日表示処理
+puts "Su Mo Tu We Th Fr Sa"
+
+#日付表示処理
+week = ""
+(start_date.mday.to_i..last_date.mday.to_i).each do |date|
+  if Date.new(year, month, date).wday == 0
```

Comment:
DateのRangeを使えば、
```suggestion
(start_date..last_date).each do |date|
  if date.wday == 0
```
でいけると思います〜

Reply:
[c6b7fdd](https://github.com/doitumura/ruby-practices/pull/2/commits/c6b7fdd05be5b43ab3977ecb023a45d34e7f8213)で修正しました。